release notes/v0.9.0.md

```diff
@@ -0,0 +1,111 @@
+k6-browser `v0.9.0` is here 🎉! This release includes:
+
+- A few breaking changes.
+- `browserContext.addCookies` has now been implemented.
+- `browserType.Connect` has now been implemented and so k6-browser can connect to an already running Chrome/Chromium browser instance.
+- Web vitals are natively supported by the k6-module.
+- Lots of fixes and refactors.
+
+
+## Breaking changes
+
+- [xk6-browser#844](https://github.com/grafana/xk6-browser/pull/844) Remove the exported `version` param from the root module. We want uses to reference the k6 version now instead of the k6-browser version.
```

Comment:
```suggestion
- [xk6-browser#844](https://github.com/grafana/xk6-browser/pull/844) Remove the exported `version` param from the root module. We want users to reference the k6 version now instead of the k6-browser version.
```
And maybe I would write it in another way such as:
```suggestion
- [xk6-browser#844](https://github.com/grafana/xk6-browser/pull/844) Remove the exported `version` param from the root module. Users should from now on reference the k6 version instead of the k6-browser version.
```